packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md

```diff
@@ -1,48 +1,43 @@
 ---
-title: Request an access token
+title: Get an access token and make a request
 ---
-After you have the following, you can request an access token:
+After you have the following, you can get an access token and make a request to an endpoint.
 
-* An Okta OIDC or OAuth 2.0 Service app
+* An Okta OpenID Connect or OAuth 2.0 Service app
 * One or more grants associated with that app
 * Users with appropriate permissions associated with the app
 
 > **Note:** Using a Service app? See <GuideLink link="../get-access-token-using-service-app">Get an access token using a Service app</GuideLink> for the steps.
 
-1. Request an access token by making a request to your Okta [Org Authorization Server's](/docs/concepts/auth-servers) `/authorize` endpoint. For testing purposes, we recommend that you use a private browser window and the Implicit grant flow. Using the Implicit grant flow streamlines authentication for testing by returning a token without introducing any additional steps. In a production environment, we recommend that you always use the Authorization Code grant flow. See [Implement the Authorization Code Flow](/docs/guides/implement-auth-code/) for more information on this grant type.
+Request an access token by making a request to your Okta [Org Authorization Server](/docs/concepts/auth-servers) `/authorize` endpoint. Only the Org Authorization Server can mint access tokens that contain Okta API scopes.
```

Comment:
Does the link not need to have the forward slash at the end `(/docs/concepts/auth-servers/)`?